tracing/providers/opentelemetry/src/main/java/io/helidon/tracing/providers/opentelemetry/OpenTelemetryTracerProvider.java

```diff
@@ -90,10 +97,17 @@ public static Tracer globalTracer() {
      * @return optional of the current span
      */
     public static Optional<Span> activeSpan() {
-        // OTel returns a no-op span if there is no current one.
-        io.opentelemetry.api.trace.Span otelSpan = io.opentelemetry.api.trace.Span.current();
+        io.opentelemetry.context.Context otelContext = io.opentelemetry.context.Context.current();
```

Comment:
I think this entire method can be just:
```
return Optional.ofNullable(io.opentelemetry.api.trace.Span.fromContextOrNull(io.opentelemetry.context.Context.current()))
  .map(otelSpan -> HelidonOpenTelemetry.create(otelSpan, io.opentelemetry.api.baggage.Baggage.current());
```
(Some `static` `import`s would make this a little easier, maybe.) What do you think?